### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by Ariel-Isaacm**

### CODE REVIEW

File: docs/watch-examples.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ Likely because of an enthusiastic pattern trying to watch thousands of files. Su
 Another reason if you're watching a large number of files could be the low default `interval`. Try increasing with `options: { interval: 5007 }`. Please see issues [#35](https://github.com/gruntjs/grunt-contrib-watch/issues/35) and [#145](https://github.com/gruntjs/grunt-contrib-watch/issues/145) for more information.
 
 ## Why spawn as child processes as a default?
-The goal of this watch task is as files are changed, run tasks as if they were triggered by the user themself. Each time a user runs `grunt` a process is spawned and tasks are ran in succession. In an effort to keep the experience consistent and continually produce expected results, this watch task spawns tasks as child processes by default.
+The goal of this watch task is as files are changed, run tasks as if they were triggered by the user himself. Each time a user runs `grunt` a process is spawned and tasks are ran in succession. In an effort to keep the experience consistent and continually produce expected results, this watch task spawns tasks as child processes by default.
```
Comment (vladikoff) [inline docs/watch-examples.md]: please include both genders here

Comment (Ariel-Isaacm) [inline docs/watch-examples.md]: done

**Comment by XhmikosR** [comment]
@Ariel-Isaacm: please squash the patches.

File: docs/watch-examples.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ Likely because of an enthusiastic pattern trying to watch thousands of files. Su
 Another reason if you're watching a large number of files could be the low default `interval`. Try increasing with `options: { interval: 5007 }`. Please see issues [#35](https://github.com/gruntjs/grunt-contrib-watch/issues/35) and [#145](https://github.com/gruntjs/grunt-contrib-watch/issues/145) for more information.
 
 ## Why spawn as child processes as a default?
-The goal of this watch task is as files are changed, run tasks as if they were triggered by the user themself. Each time a user runs `grunt` a process is spawned and tasks are ran in succession. In an effort to keep the experience consistent and continually produce expected results, this watch task spawns tasks as child processes by default.
```
Comment (eddiemonge) [inline docs/watch-examples.md]: This seems correct. The grammar before this change could use some work though. I'd get rid of `The goal of this watch task is`.

**Comment by XhmikosR** [comment]
@Ariel-Isaacm: that's not a rebase.

**Comment by Ariel-Isaacm** [comment]
I think i'm doing a mess :disappointed:

**Comment by XhmikosR** [comment]
`git rebase -i upstream/master`

**Comment by Ariel-Isaacm** [comment]
Done thanks :)!

**Comment by XhmikosR** [comment]
Almost there :P One more time to fix your commit message.

**Comment by Ariel-Isaacm** [comment]
How about now? thanks for being so patient :)

**Comment by XhmikosR** [comment]
Now it just shows "All commits Squashed" which doesn't describe what the commit is for.

**Comment by Ariel-Isaacm** [comment]
I changed it to "Fix spelling"

**Pull request merged**